docker-compose.yml

```diff
@@ -45,13 +49,18 @@ services:
             # BEHAVIOR_INBOUNDTTL TODO
             - BEHAVIOR_CTXPROPAGATION=ctxclient,ctxserver,transport
             - BEHAVIOR_APACHETHRIFT=apachethriftclient,apachethriftserver
+            - BEHAVIOR_ONEWAY=client_oneway,server_oneway,transport_oneway,encoding
+            - BEHAVIOR_ONEWAYCTXPROPAGATION=client_oneway,server_oneway,transport_oneway
```

Comment:
Hard to gauge what the rules evident are for naming these things, but I recommend another underscore beteen ONEWAY_CTXPROPAGATION, treating ONEWAY and CTXPROPAGATION as single words as above, and use hyphen in lower-case values, per the precedent established by python-sync.